NOpenType/Glyph.cs

```diff
@@ -21,24 +32,31 @@ public Point(short x, short y, bool on)
         public short Y { get { return _y; } }
         public bool On { get { return _on; } }
     }
-    
+
     public class Glyph
     {
         private readonly short[] _x;
         private readonly short[] _y;
         private readonly bool[] _on;
         private readonly ushort[] _contourEndPoints;
         private readonly Bounds _bounds;
+        private Flag[] _flags;
```

Comment:
There is no need to store the flags. Only the "on" property is relevant. The other parts of the flags is only for parsing ttf.